cpanfile

```diff
@@ -115,8 +115,7 @@ on 'test' => sub {
 
 };
 
-on 'devel' => sub {
-    requires 'Perl::Tidy', '== 20230912.0.0';
```

Comment:
I don't know why it removed Perl::Tidy here.
I really hope next time we can do everything in smaller steps.
The usage of update-deps changed, but I wasn't aware of this.
I will debug now

Reply:
Ah, I forgot to rename the target to develop here. Should be fixed now